docs/contributtion/feature-flags.md

```diff
@@ -0,0 +1,67 @@
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The Spirit Design System codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in React, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
+how to configure it. Folders and stories within this section in Storybook show
+components with all feature flags turned on.
+
+## Current feature flags
+
+Feature flags prefixed with `enable-*` contain features we'd like consuming projects to
+test and give us feedback on. It will be made by default in the version specified in the name.
+In most of the cases these features will be breaking changes - logically or visually.
+They're generally stable and unlikely to change but may change based on your feedback.
+We just do not want to break your application or design without your consent.
+
+Flags prefixed with `enable-v#-*` will be marked as `true` or "on" by default in the next major version.
+
+For more details on this approach, see the
+[feature flag documentation][feature-flag-documentation]
+in the Spirit Design System monorepo.
+
+| Flag | Description | Default | SASS flag |
```

Comment:
somewhere you mention we will document feature flags in the main readme. Why do we need this table here then?

Reply:
I have reword it "somewhere" to point it to this one an only place, where it should be documented.